device.yml

```diff
@@ -0,0 +1,407 @@
+  EncoderSampling:
+    address: 91
+    type: U8
+    access: Write
+    maskType: EncoderSamplingRate
+    description: Sets the sampling rate of the magnetic encoder.
+  Reserved2:
+    <<: *reserved
+    address: 92
+  Reserved3:
+    <<: *reserved
+    address: 93
+  ServoPeriod:
+    address: 94
+    type: U16
+    access: Write
+    description: Sets the period of the servo pulses (us).
+  Servo0PulseWidth: &servopulsewidth
+    address: 95
+    type: U16
+    access: Write
+    description: Sets the pulse width of the Servo0 pulses (us).
+  Servo1PulseWidth:
+    <<: *servopulsewidth
+    address: 96
+    description: Sets the pulse width of the Servo1 pulses (us).
+  Servo2PulseWidth:
+    <<: *servopulsewidth
+    address: 97
+    description: Sets the pulse width of the Servo2 pulses (us).
+  Reserved4:
+    <<: *reserved
+    address: 98
+  Reserved5:
+    <<: *reserved
+    address: 99
+  OpticalFlow:
+    address: 100
+    type: S16
+    access: Event
+    description: Generated event with the latest read from the optical flow sensor.
+bitMasks:
+  AuxiliaryInput:
+    bits:
+      Aux0: 0x1
+      Aux1: 0x2
+      Aux0Changed: 0x20
+      Aux1Changed: 0x40
+    description: Specifies the state of auxiliary input lines.
+  DigitalOutputs:
+    description: Specifies the available digital output lines.
+    bits:
+      Out0: 0x1
+      Out1: 0x2
+      Out2: 0x4
+      Out3: 0x8
+      Out4: 0x10
+      Out5: 0x20
+      Out6: 0x40
+      Out7: 0x80
+      Out8: 0x100
+      Out9: 0x200
+  PwmAndStimMapping:
+    description: Specifies the mapping PWM and Stimulation to digital output lines.
+    bits:
+      Pwm0ToOut1: 0x1
+      Pwm0ToOut2: 0x2
+      Pwm0ToOut3: 0x4
+      Pwm1ToOut6: 0x8
+      Pwm1ToOut7: 0x10
+      Pwm1ToOut8: 0x20
+      Pwm2ToOut9: 0x40
+      Stim0ToOut0: 0x80
+      Stim0ToOut5: 0x100
+  Pwm:
+    description: Specifies the available PWM protocols.
+    bits:
+      Pwm0: 0x1
+      Pwm1: 0x2
+      Pwm2: 0x4
+  Stim:
+    description: Specifies the available Stim protocols.
+    bits:
+      Stim0: 0x1
+groupMasks:
+  AcquisitionMode:
+    description: Available configurations for a PWM line.
+    values:
+      Continuous: 0
+      Finite: 1
+  TriggerSources:
+    description: Available trigger sources.
+    values:
+      Software: 0
+      Aux0Rising: 1
+      Aux0Falling: 2
+      Aux0WhileHigh: 3
+      Aux0WhileLow: 4
+      Aux1Rising: 16
+      Aux1Falling: 32
+      Aux1WhileHigh: 64
+      Aux1WhileLow: 128
+  EnableFlag:
+    description: The state of a configuration.
+    values:
+      Disabled: 0
+      Enabled: 1
+  ExpansionBoardTypes:
+    description: Specifies the available expansion boards implemented.
+    values:
+      Breakout: 0
+      Encoder: 1
+      ServoMotor1: 2
+      ServoMotor2: 3
+      ServoMotor3: 4
+      OpticalFlow: 5
+  EncoderSamplingRate:
+    description: Specifies the sampling rate of the encoder.
+    values:
+      At50Hz: 0
```

Comment:
Consider alternatives to `At`:
  - [x] `SamplingRate50Hz`
  - [ ] `Sampling50Hz`
  - [ ] `Fs50Hz` / `FS50Hz`